_includes/header.html

```diff
@@ -39,7 +39,7 @@
 							<li>
 								<a href="{{ baseurl }}./#welcome">Welcome</a>
 							</li>
-     						<li>
+							<li>
```

Comment:
Remove whitespace

Reply:
Why this one? The code is properly indented here.

Reply:
This is fine.